src/app/pages/CreateConnectorPage/StepConfigurator.tsx

```diff
@@ -104,6 +104,32 @@ const ConnectedJsonSchemaConfigurator: FunctionComponent<{
   );
 };
 
+export const ConfiguratorCustomStepDescription: FunctionComponent<{
+  actor: ConfiguratorActorRef;
+}> = ({ actor }) => {
+  const { t } = useTranslation();
+  let {
```

Comment:
`const` would be better, also if you don't need `configuration` or `connector` you can just delete line 113/114, no need to prepend with `_`.  Then you should also be able to get rid of line 121 and 119, if you don't need those values, you don't need to copy them from the state in the `useSelector` call.